### PR TITLE
chore: allow times timeline update, while execution in progress #patch

### PR DESCRIPTION
**Pull request by anrusina**
This PR adds few fixes for not yet released feature: timeline view.
- Allow to see proper execution time for child items in timeline view
- Allow to dynamically update tasks timeline, if execution is still in progress
- remove start/end nodes for branchNode tasks
- Fix for proper node naming for all types of nesting
- Added dynamic timeline scale update

basic walk through: https://share.getcloudapp.com/geud604n

## Type
 - [X] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [X] Code completed
 - [X] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description

Timeline feature is still under feature flag, so won't be yet visible for end user.
As NodeExecution provider information has been changed, I ensured to test Nodes view for all type of node executions.
Next execution were used for testing of Nodes and Timeline tabs

**Standard task:**
[X] https://localhost.demo.nuclyde.io:3000/projects/flytesnacks/domains/development/executions/rnktdb3skr?duration=all
[X] https://localhost.demo.nuclyde.io:3000/projects/flytesnacks/domains/development/executions/ubcn6jgs59?duration=all

**Branch** 
[X] https://localhost.demo.nuclyde.io:3000/projects/flytesnacks/domains/development/executions/waekf4jd2d?duration=all
- Weird issue on clicking to open info for task. - we already saw that error

**Dynamic**
[X] https://localhost.demo.nuclyde.io:3000/projects/flytesnacks/domains/development/executions/mjnv9tfbd5?duration=all
The Timeline view is currently doesn't work as expected for Dynamic executions. It shows up top level dynamic node, and doesn't rendere any child tasks in timeline view.

**Dynamic + Branch**
[X] https://localhost.demo.nuclyde.io:3000/projects/flytesnacks/domains/development/executions/zalmx8wnp3?duration=all 

## Tracking Issue
Partial work for: flyteorg/flyteconsole#302
Partial fix for : https://github.com/flyteorg/flyteconsole/issues/279 - the bugs are removed, however infrastructure clean-up is still would be needed.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
+    // temporarily disabled
+    '@typescript-eslint/no-var-requires': 'off', // 27
+    '@typescript-eslint/ban-types': 'warn', // 62
+    '@typescript-eslint/no-empty-function': 'warn', // 60
+    '@typescript-eslint/explicit-module-boundary-types': 'off', // 296
+    '@typescript-eslint/no-explicit-any': 'warn', // 118
+    '@typescript-eslint/no-non-null-assertion': 'warn', // 59
+    'react-hooks/exhaustive-deps': 'warn' // 24
+  },
+  overrides: [
```
Comment (anrusina) [inline .eslintrc.js:33]: Only this part is added - to allow **any** usage in sotrybook and test files

File: jest.config.js [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
+    '__stories__',
+    '<rootDir>/.storybook',
+    '<rootDir>/node_modules',
+    '<rootDir>/dist',
+    '<rootDir>/build',
+    '<rootDir>/src/tsd',
+    '<rootDir>/.eslintrc.js',
+    '\\.config.js$'
+  ],
+  coverageReporters: ['text', 'json', 'html'],
```
Comment (anrusina) [inline jest.config.js:38]: add coverage report in html to simplify how the reports looks locally

**Review by jsonporter — APPROVED** [review]
lgtm

**Review by jsonporter — APPROVED** [review]
lgtm

**Pull request merged**